shared/middleware/helpers/table.rb

```diff
@@ -74,7 +74,7 @@ def update(id, value, ip_address)
 
   def to_a()
     items.map do |row|
-      JSON.load(row[:value]).merge(id: row[:row_id])
+      JSON.load(row[:value]).merge('id' => row[:row_id])
```

Comment:
What is the intent of this change?  (I'll be swapping out this table implementation so I should make sure I preserve any important behavior.)